sdk/python/symbolchain/TransactionDescriptorProcessor.py

```diff
@@ -22,7 +22,7 @@ def _lookup_value_and_apply_type_hints(self, key):
 	def lookup_value(self, key):
 		"""Looks up the value for key."""
 		value = self._lookup_value_and_apply_type_hints(key)
-		if not any(isinstance(value, type_class) for type_class in [str, bytes, tuple]) and hasattr(value, '__iter__'):
+		if not any(isinstance(value, type_class) for type_class in [str, bytes, tuple]) and isinstance(value, list):
```

Comment:
can you please add regression test to TransactionDescriptorProcessorTest ?

Reply:
hmm, I think the use of `hasattr` here was because we wanted to also handle tuples - but if tests are passing, I'm ok with this.
(and in general any kind of iterators)